RelayTestApp/Assets/Scripts/Managers/BrainCloudManager.cs

```diff
@@ -133,6 +134,11 @@ void LogErrorThenPopUpWindow(int status, int reasonCode, string jsonError, objec
     {
         if (m_dead) return;
 
+        if (jsonError.Contains("Disconnected by server from end match message"))
```

Comment:
Is it possible to check this in a different way? A string that looks exactly like this could change or might not exist in future versions.